tox.ini

```diff
@@ -0,0 +1,22 @@
+[tox]
+envlist = py35,py36
+
+[testenv]
+deps =  .[dev]
+        icontract==1.5.9
+        typing_extensions==3.6.6
+
+changedir = {envtmpdir}
+commands =
+    python3 {toxinidir}/precommit.py
+
+setenv =
+    COVERAGE_FILE={envbindir}/.coverage
+
+[testenv:pex]
```

Comment:
I tried this as well and it failed. The failure message was different this time. It said it can't import icontract. I commented icontract in the code and afterwards tox created a working .pex file without any errors. Can we look at this together on Friday?